controllers/k6_start.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func isServiceReady(log logr.Logger, service *v1.Service) bool {
-	resp, err := http.Get(fmt.Sprintf("http://%v.%v.svc.cluster.local:6565/v1/status", service.ObjectMeta.Name, service.ObjectMeta.Namespace))
+	resp, err := http.Get(fmt.Sprintf("http://%v.%v.svc:6565/v1/status", service.ObjectMeta.Name, service.ObjectMeta.Namespace))
```

Comment:
Could you please remove this one? TBH, I'm wary of this change given the issues described in #186 Let's keep the changes in the PR straight-forward.

Reply:
done